docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -2,7 +2,7 @@ summary: Learn how to crop and resize images in templates and PHP code
 
 # Image
 
-Image files can be stored either through the `[Image](api:SilverStripe\Assets\Image)` dataobject, or though [DBFile](api:SilverStripe\Assets\Storage\DBFile) fields.
+Image files can be stored either through the [Image](api:SilverStripe\Assets\Image) dataobject, or though [DBFile](api:SilverStripe\Assets\Storage\DBFile) fields.
```

Comment:
the change should be:

```diff
-`[Image](api:SilverStripe\Assets\Image)`
+[`Image`](api:SilverStripe\Assets\Image)
```

Reply:
Oh, I didn't know the backticks could be inside the link label!